tests/_support/Page/Acceptance/Administrator/ControlPanelPage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Page\Acceptance\Administrator;
+
+use Page\Acceptance\Administrator\AdminPage;
+
+class ControlPanelPage extends AdminPage
+{
+    public static $url = "/administrator/index.php";
+
+    public static $pageTitle = 'Control Panel';
+
+    public static $pageTitleContext = ['class' => 'page-title'];
```

Comment:
@javigomez not sure if i understand you.

```php
public static $usefulAndShortVariableName = [
   'articleTitleField' => ['id' => 'jform_title'],
   // ...
];

$I->fillField(Article::_('articleTitleField'), $title); 
```

You can have an array in an array. Or do you mean something different?